processor/probabilisticsamplerprocessor/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/processor/probabilisticsamplerprocessor
+
+go 1.16
+
+require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.0.0-20210820200022-94ff24b8ebbd
```

Comment:
Use version 0.0.0-0000 or whatever we do in other scenarios and a replace statement for the module to get the local